### PR TITLE
Feature/tpp 00046

### LINKED ISSUE #46: TPP-00046: Send Daily Notifications
> As a user of Menstruation Nation, I want to be able to receive daily notifications reminding me to log my symptoms

**Technical Information:**
- by default for now, let's send daily reminder notifications at 9am 
- https://github.com/react-native-push-notification/ios will likely be helpful
- At 9am, send a notifications. Use the blank template specifications below
- Tapping on notification will take you to M. Nation app's calendar month view 

**Styling Requirements:**
Settings notification page
Screen Shot 2021-12-22 at 2 20 21 PM
- Add text "Every day at 9:00 am" underneath "Remind me to log symptoms " LOL sorry don't follow the image follow me   

Blank ios notification 
Screen Shot 2021-12-22 at 2 22 39 PM
- the app icon should be the menstruation nation badge (same as app icon when on home screen)
Screen Shot 2021-12-22 at 2 26 21 PM
- instead of "EXAMPLE", should say "MENSTRUATION NATION" 
- title: Daily log reminder
- no subtitle 
- body: "Daily reminder to log your symptoms!"

### PR DESCRIPTION
**Pull request by edwardhuahan**
# Description

Added toggle to turn on daily notifications on 9 am.
When these notifications are clicked, it will open the calendar view.
Screen Shot 2022-01-20 at 22 35 27
Screen Shot 2022-01-20 at 22 35 06

Fixes #46 

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings

### CODE REVIEW

File: tpp-app/App.js [hunk 1]
```diff
@@ -96,15 +108,48 @@ export function MyTabs() {
 
 
 export default function App() {
+  const navigationRef = useNavigationContainerRef();
+
+  // Requests for notification permissions and also creates a local notification listener
+  // Does not handle remote notifications from a server.
+  useEffect(() => {
+
+    PushNotificationIOS.addEventListener('localNotification', ((notification) => {
```
Comment (kenneth-miura) [inline tpp-app/App.js:102]: isn't the event listener here identical to the previously defined `onLocalNotification`? If it is you should just replace it with the function u defined earlier

File: tpp-app/src/settings/Settings.js [hunk 2]
```diff
@@ -78,14 +83,44 @@ const Preferences = (props) => {
...
+                    hour: true,
+                    minute: true,
+                },
+            });
+        } else {
+            PushNotificationIOS.removePendingNotificationRequests(['remindsymptoms'])
+        }
+    }
+
+    const getCorrectDate = () => {
```
Comment (kenneth-miura) [inline tpp-app/src/settings/Settings.js:116]: Not sure if the comment out on the line `fireDate: getCorrectDate()` is on purpose. If it is, you may as well delete the commented-out line, as well as `getCorrectDate`.

**Review by kenneth-miura — CHANGES_REQUESTED** [review]
Really like the comments! Helped a lot in navigating usage of a library I've never touched. I have a few comments, but they're pretty small. Great work!

**Review by annie951977 — APPROVED** [review]
The comments really help!

**Review by kenneth-miura — APPROVED** [review]
don't remember what my comments were but I'm assuming u resolved them lol gj

**Pull request merged**